lxd/storage/backend_lxd.go

```diff
@@ -1270,7 +1270,7 @@ func (b *lxdBackend) RefreshCustomVolume(projectName string, srcProjectName stri
 			srcVolStorageName := project.StorageVolume(srcProjectName, srcVolName)
 			srcVol := srcPool.GetVolume(drivers.VolumeTypeCustom, contentType, srcVolStorageName, srcVolRow.Config)
 
-			srcVol.MountTask(func(mountPath string, op *operations.Operation) error {
+			err = srcVol.MountTask(func(mountPath string, op *operations.Operation) error {
```

Comment:
Good catch!

Reply:
I'm working in this area so will fix it in my PR as well to reduce the size of this one.